setup.py

```diff
@@ -1,27 +1,51 @@
 """
 distutils installation script for pyFreenet
 """
-import sys, os
+import sys
+import os
+import logging
+import distutils.command.install
+from distutils.core import setup
+
+
 
 doze = sys.platform.lower().startswith("win")
 
 scripts = ["freesitemgr", "pyNodeConfig", 
            "fcpget", "fcpput", "fcpupload", "fcpgenkey", "fcpinvertkey", "fcpredirect", "fcpnames", 
-           "fproxyproxy"# , "freedisk"  # <- not yet reviewed
+           "fproxyproxy", "copyweb"  # , "freedisk"  # <- not yet reviewed
            ]
 if doze:
     for i in range(len(scripts)):
         scripts[i] += ".py"
 
-from distutils.core import setup
+
+class pyfreenet_install(distutils.command.install.install):
+    def run(self, *args, **kwds):
+        distutils.command.install.install.run(self, *args, **kwds)
+        man_dir = os.path.abspath("./manpages/")
+        man_target_dir = os.path.join(self.install_base, "share/man/man1")
+        try:
+            print "Creating man-page directory at", man_target_dir
+            os.makedirs(man_target_dir)
+        except Exception as e:
+            if str(e).endswith("File exists: '" + man_target_dir + "'"):
```

Comment:
This should not rely on the string. Would it be reasonable to handle `OSError` and check that `e.errno == errno.EEXIST`?